### DISCUSSION TITLE
Creating a notification when a stream goes live

### DISCUSSION THREAD

**Discussion by racaroni** [TwitchIO/TwitchIO#375] [Questions] [answered]
Total programming newbie here.
Could some one provide example code of if streamer goes live, print "{streamer] is live!"?

**Comment by chillymosh** [comment, marked as answer]
Hi @racaroni,

We have an example of how to do this in our eventsub example.
https://twitchio.dev/en/latest/exts/eventsub.html#running-eventsub-inside-a-commands-bot

Be aware that this does currently require a publicly accessible URL, non self-signed SSL cert and a reverse proxy, such as NGINX.

You will just need to adjust the subscription method and callback event to the desired ones.

```
await esclient.subscribe_channel_stream_start(broadcaster=123456789)

@esbot.event()
async def event_eventsub_notification_stream_start(payload: eventsub.StreamOnlineData) -> None:
    channel_name =payload.data.broadcaster.name
    print(f"{channel_name } ONLINE")
```

**Reply by racaroni** [reply]
Thank you!
